Makefile

```diff
@@ -7,37 +7,37 @@ docker-nearcore-nightly:
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p near-vm-runner-standalone --features performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p state-viewer --features performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p store-validator --features performance_stats,memory_stats
 
 nightly-release:
-	CARGO_PROFILE_RELEASE_DEBUG=true cargo $(RUST_OPTIONS) build -p neard --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
-	cargo $(RUST_OPTIONS) build -p near-vm-runner-standalone --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
-	cargo $(RUST_OPTIONS) build -p state-viewer --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
-	cargo $(RUST_OPTIONS) build -p store-validator --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' CARGO_PROFILE_RELEASE_DEBUG=true cargo $(RUST_OPTIONS) build -p neard --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p near-vm-runner-standalone --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p state-viewer --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p store-validator --release --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
 
 nightly-debug:
-	CARGO_PROFILE_RELEASE_DEBUG=true cargo $(RUST_OPTIONS) build -p neard --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
-	cargo $(RUST_OPTIONS) build -p near-vm-runner-standalone --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
-	cargo $(RUST_OPTIONS) build -p state-viewer --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
-	cargo $(RUST_OPTIONS) build -p store-validator --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' CARGO_PROFILE_RELEASE_DEBUG=true cargo $(RUST_OPTIONS) build -p neard --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p near-vm-runner-standalone --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p state-viewer --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
+	RUSTFLAGS='-D warnings' cargo $(RUST_OPTIONS) build -p store-validator --features nightly_protocol,nightly_protocol_features,performance_stats,memory_stats
```

Comment:
Can we specify it just once? Looks like it is possible for make:

```
19:39:14|~/tmp
λ bat -p Makefile 
export MY_VAR = "bar"

task:
    env

19:39:21|~/tmp
λ make | rg MY_VAR
MY_VAR="bar"
```

not sure if there's an equivalent for buildkite.